app/org/sagebionetworks/bridge/services/ExternalIdService.java

```diff
@@ -73,19 +73,15 @@ public void reserveExternalId(Study study, String externalIdentifier) {
         checkNotNull(study);
         checkArgument(isNotBlank(externalIdentifier));
 
-        if (study.isExternalIdValidationEnabled()) {
-            externalIdDao.reserveExternalId(study.getStudyIdentifier(), externalIdentifier);
-        }
+        externalIdDao.reserveExternalId(study.getStudyIdentifier(), externalIdentifier);
     }
     
     public void assignExternalId(Study study, String externalIdentifier, String healthCode) {
         checkNotNull(study);
         checkArgument(isNotBlank(externalIdentifier));
         checkArgument(isNotBlank(healthCode));
         
-        if (study.isExternalIdValidationEnabled()) {
-            externalIdDao.assignExternalId(study.getStudyIdentifier(), externalIdentifier, healthCode);
-        }
+        externalIdDao.assignExternalId(study.getStudyIdentifier(), externalIdentifier, healthCode);
```

Comment:
It would be safest to add this back, and check redundantly in the ParticipantService to verify the functionality is not executed in the tests.